scalafmt-cli/src/main/scala/org/scalafmt/cli/CliOptions.scala

```diff
@@ -54,7 +54,7 @@ object CliOptions {
         out =
           guardPrintStream(parsed.quiet && !parsed.stdIn)(parsed.common.out),
         info = guardPrintStream(
-          parsed.quiet || parsed.writeMode == WriteMode.List
+          parsed.stdIn || parsed.quiet || parsed.writeMode == WriteMode.List
```

Comment:
is it important that one is reading from stdin? what if you are reading from a file but writing to stdout (via `WriteMode.StdOut`?)

Reply:
Good point, as far as I understand it if I consume from `stdin` the result will always be written to `stdout` which is problematic when writing info to stderr as it can break pipelines, but the same is true when consuming from a file and writing to `stdout`. I'll change this.